tools/report-converter/tests/unit/cpplint_output_test_files/files/sample.cpp

```diff
@@ -0,0 +1,264 @@
+// via https://github.com/cpplint/cpplint/blob/develop/samples/silly-sample/src/sillycode.cpp
```

Comment:
It would be better to not copy file content from other repositories to avoid license problems. So I recommend to create our own example file which contains 2-3 errors reported by cpplint.